frontend/src/queries/schema.ts

```diff
@@ -765,6 +776,11 @@ export interface InsightPersonsQuery {
     source: InsightQueryNode
     day?: string
     status?: string
+    /**
+     * An interval selected out of available intervals in source query
+     * @asType integer
+     */
+    selected_interval?: number
```

Comment:
I wonder if in the context of this query, `selected_` is redundant? Both "day" and "status" above are also technically "selected".

Might be worth simplifying to just `interval` while we still can?

Reply:
Uh yeah I like that.